### PR TITLE
MGMT-12977: MGMT-13329: Assisted-Installer: Update master release configurations file to point to 4.13

### PR DESCRIPTION
**Pull request by eliorerz**
Also Remove e2e-metal-assisted-4-13 periodic job

/cc @osherdp @adriengentil

### CODE REVIEW

**Comment by eliorerz** [comment]
/hold wait for 

- https://github.com/openshift/assisted-service/pull/4898

- https://github.com/openshift/assisted-image-service/pull/111

**Comment by adriengentil** [comment]
@eliorerz  @osherdp before adding more entropy in our CI, can we wait for https://github.com/openshift/assisted-test-infra/pull/1989 to merged and for the issue about the operator that fails to be pulled to be clarified?

**Comment by osherdp** [comment]
> @eliorerz @osherdp before adding more entropy in our CI, can we wait for [openshift/assisted-test-infra#1989](https://github.com/openshift/assisted-test-infra/pull/1989) to merged and for the issue about the operator that fails to be pulled to be clarified?

Yes good idea

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml [hunk 1]
```diff
@@ -852,15 +856,6 @@ tests:
         OPENSHIFT_VERSION=4.12
         OPENSHIFT_INSTALL_RELEASE_IMAGE=""
     workflow: assisted-baremetal
-- as: e2e-metal-assisted-4-13
```
Comment (osherdp) [inline ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:855]: maybe we should keep this job only for the ``bump_ocp_upstream.py`` script that triggers the affected jobs (it doesn't have any handling for a default version, it just do ``/test edge-e2e-metal-assisted-4-Y``)

**Review by osherdp — CHANGES_REQUESTED** [review]

**Comment by adriengentil** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-metal-assisted-day2-arm-worker-periodic

**Comment by adriengentil** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-day2-arm-workers-periodic

**Comment by adriengentil** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-metal-assisted-kube-api-late-binding-single-node-periodic

**Comment by adriengentil** [comment]
> /test ci/rehearse/periodic-ci-openshift-assisted-test-infra-master-e2e-vsphere-assisted

/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-edge-e2e-vsphere-assisted-periodic

**Comment by adriengentil** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-day2-arm-workers-periodic
flake :thinking:

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-installer-master-edge-e2e-ai-operator-ztp

**Comment by eliorerz** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-capi-periodic
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-sno-day2-workers-periodic
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-sno-day2-workers-late-binding-periodic
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-periodic
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-hypershift-zero-nodes-periodic

**Comment by eliorerz** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-hypershift-zero-nodes-periodic

**Comment by eliorerz** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-capi-periodic

**Comment by eliorerz** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-sno-day2-workers-periodic

**Comment by eliorerz** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-sno-day2-workers-late-binding-periodic

**Comment by eliorerz** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-periodic

**Comment by eliorerz** [comment]
/pj-rehearse pull-ci-openshift-assisted-image-service-main-edge-e2e-ai-operator-ztp

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml (hunk 1 above)
Comment (eliorerz) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml]: Done

**Comment by gamli75** [comment]
LGTM

**Review by gamli75 — APPROVED** [review]

**Pull request merged**